### PR TITLE
Adding fwdLogs feature info

### PR DESCRIPTION
**Pull request by hegderohit89**

### CODE REVIEW

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 1]
```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
     Tls          On
 ```
 
+==== Configure Log Forwarding
```
Comment (hegderohit89) [inline modules/ROOT/pages/runtime-fabric-logs.adoc:246]: Should we say  `Configure Log Forwarding From` ?
As this section is to enable/disable what logs needs to be forwarded.  @sundippatel

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 2]
```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
...
+
+The following configuration options are available:
+
+* Mule Applications: Forward logs from Mule applications.
+* Internal load balancer and component logs: Forward logs from Runtime Fabric namespace compenents and internal load balancers.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: Suggest removing "namespace", and make balancers singular.

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 3]
```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
...
+* Runtime Fabric appliance: Forward logs from Anypint Monitoring and Kube-system namespaces.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: Spelling on Anypoint.
"Kube-system namespaces" -> "appliance services"

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 4]
```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
...
+
+[NOTE]
+====
+* By default all options on Forward logs configuration are checked.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: "All options are enabled by default."

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 5]
```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
...
+* Configuring Forward logs from Mule applications is not currently supported, so this option is checked and disabled. You can still enable or disable per application log forwarding from Applications page.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: "Disabling log forwarding for Mule applications is not supported. Log forwarding can be disabled per application during deployment."

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 6]
```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
...
+====
+
 === Kubernetes Metadata
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-fabric-logs.adoc:260]: @hegderohit89 Any problem if we rename "Kubernetes" to "Appliance"?

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 7]
```diff
@@ -282,7 +296,7 @@ You can send a test message to verify connectivity between Runtime Fabric and th
 .. Trigger a test message for the selected service.
 .. Verify the text that is forwarded to log forwarding solution.
 
-== Disable Anypoint Monitoring or Output Plugin Log Forwarding
+== Disable Anypoint Monitoring or External Log Forwarding per Application
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-fabric-logs.adoc:299]: I think we're missing a step here where we need the user to deploy or re-deploy their application.
@hegderohit89 to confirm.

**Review by sundippatel — COMMENTED** [review]
Left in-line comments.

File: modules/ROOT/pages/runtime-fabric-logs.adoc (hunk 6 above)
Comment (hegderohit89) [reply modules/ROOT/pages/runtime-fabric-logs.adoc:260]: This is the metadata available at external service providers for users to perform search, so i feel its better to keep it Kubernetes. As most of the collectors have kubernetes.* as metadata search filters

File: modules/ROOT/pages/runtime-fabric-logs.adoc (hunk 7 above)
Comment (hegderohit89) [reply modules/ROOT/pages/runtime-fabric-logs.adoc:299]: Ya thats right, i will add that

**Review by paigehargrave — APPROVED** [review]

**Review by paigehargrave — APPROVED** [review]

**Pull request merged**